tools/install-unprivileged.sh

```diff
@@ -202,15 +203,24 @@ latesturl()
 		LASTURL="$URL"
 		LASTPKGS="$(curl -Ls "$URL")"
 	fi
+	if [ $RETRY -gt 0 ]; then
+		LASTPKGS="$(curl -Ls "$URL")"
+	fi
 	typeset LATEST="$(echo "$LASTPKGS"|sed -e 's/.*href="//;s/".*//' -e 's/\.mirrorlist//' -e 's/\-32bit//' -e 's@^\.\/@@' |grep "^$2-[0-9].*$ARCH"|tail -1)"
 	if [ -n "$LATEST" ]; then
+		RETRY=0
 		echo "$URL/$LATEST"
 	elif [ "$4" = true ]; then
+		RETRY=0
 		URL="${URL/\/updates\///releases/}"
 		URL="${URL/\/Packages\///os/Packages/}"
 		latesturl "$URL" "$2" false false
 		return $?
+	elif [ $RETRY -lt 5 ]; then
+        RETRY=$((RETRY+1))
+        latesturl "$URL" "$2" "$3" "$4"
```

Comment:
These two lines should be indented with an extra tab.

Reply:
Add a `return $?` just to make sure future code changes don't let it continue to do something else further down, just like is done in the preceding elif.

Reply:
Thank you! I added `return $?`.